data/helpers.d/print

```diff
@@ -8,12 +8,11 @@ ynh_die() {
 # Display a message in the 'INFO' logging category
 #
 # usage: ynh_info "Some message"
-ynh_info()
-{
-    echo "$1" >>"$YNH_STDINFO"
+ynh_print_info() {
```

Comment:
We have already this one: https://github.com/YunoHost/yunohost/blob/debian/3.2.0-testing1/data/helpers.d/print#L11

Reply:
We discussed with Maniack and decided to go for a unified "ynh_print_<level>" (c.f. other helpers)